authentication/authentication.go

```diff
@@ -114,22 +115,25 @@ type Authentication struct {
 	OAuth        *OAuth
 	Passwordless *Passwordless
 
-	url             *url.URL
-	basePath        string
-	debug           bool
-	http            *http.Client
-	auth0ClientInfo *client.Auth0ClientInfo
-	common          manager
-	clientID        string
-	clientSecret    string
+	auth0ClientInfo   *client.Auth0ClientInfo
+	basePath          string
+	common            manager
+	clientID          string
+	clientSecret      string
+	clockTolerance    time.Duration
```

Comment:
Is `clockTolerance` only used for validating ID token claims? If so, should we prefix it with `idToken` like `idTokenSigningAlg` and `idTokenValidator`?

Reply:
Yeah it is, I will rename it